code/modules/psionics/psionic_items.dm

```diff
@@ -249,6 +284,37 @@
 		list(mode_name="armor penetrating", projectile_type=/obj/item/projectile/plasma, fire_sound='sound/weapons/Laser.ogg', fire_delay=12, icon="kill", projectile_color = "#461266"),
 	)
 
+/obj/item/gun/energy/gun/martin/cult
+	name = "\"Star\" psi-laser pistol"
+	desc = "A laser weapon formed from the mind of a psion who shaped an oddity into a weapon. A psionic copy of a martin, an oft overlooked and ignored weapon. Perhaps this is formed from the \
```

Comment:
```suggestion
	desc = "A laser weapon formed from the mind of a psion who shaped an oddity into a weapon. A psionic copy of a martin, an often overlooked and ignored weapon. Perhaps this is formed from the \
```

Reply:
"Oft" isn't a grammar mistake tho, just an archaic way of saying "often"

Reply:
I still think that 'often' sound better.

Reply:
Gonna leave it as oft since its psionics,